### ISSUE TITLE
Navigate variations

### ISSUE THREAD

**Issue by adonis28850** [franciscoBSalgueiro/en-croissant#263]
### Describe the feature

It would be nice to have something similar to Lichess or other software, where you get the option to choose between all the subvariations when you get to a point where the PGN has more than 1 possible move.

For example, in the following image you can select with the arrows/mouse between the main move (Ne3), or another variation (Nf2):

![image](https://github.com/franciscoBSalgueiro/en-croissant/assets/25471663/8adf80f6-34df-434e-9e53-95a25b711fbd)

**Comment by OleBroemelbue** [comment]
yes please! Right now pgns with more than 1 line are hard to fully understand bc the structure of the notation is also confusing.
